stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsAdapter.kt

```diff
@@ -268,9 +288,21 @@ internal class PaymentOptionsAdapter(
         }
 
         fun setSelected(selected: Boolean) {
-            binding.card.isSelected = selected
+            binding.root.isSelected = selected
             binding.checkIcon.isVisible = selected
         }
+
+        override fun setEnabled(enabled: Boolean) {
```

Comment:
These are needed in addition to the duplicateState in the layout files?

Reply:
Yes, from my tests `enabled` is not duplicated.

Reply:
What does duplicateState provide for us?

Reply:
I see the comment below about selected being propagated, but not enabled.

Reply:
Do we need to propagate the selected state?